platforms/software/antlr/src/main/java/org/gradle/api/plugins/antlr/AntlrPlugin.java

```diff
@@ -44,6 +44,7 @@
  */
 public abstract class AntlrPlugin implements Plugin<Project> {
     public static final String ANTLR_CONFIGURATION_NAME = "antlr";
+    public static final String ANTLR_TASK_GROUP = "Antlr";
```

Comment:
This property is missing javadoc, `@since` and `@Incubating`. 

Public static fields are difficult to maintain (they are difficult to deprecate, for example, since they do not execute code when the user accesses them).

So, I would instead vote to not make this property public, and instead hard-code the value in the test.

Reply:
@jvandort 
Thanks for the feedback. I’ve updated the code to remove the public property and hard-coded the value in the task instead.